.github/workflows/build-ale-py.yml

```diff
@@ -32,15 +32,19 @@ jobs:
       run: echo "ALE_BUILD_VERSION=${GITHUB_REF#refs/*/v}" >> $GITHUB_ENV
 
     - name: Build wheels
-      uses: pypa/cibuildwheel@v2.0.0
+      uses: pypa/cibuildwheel@v2.1.1
       env:
         CIBW_MANYLINUX_X86_64_IMAGE: "ghcr.io/jessefarebro/manylinux2014-vcpkg"
-        CIBW_SKIP: "*i686* *win32* pp*"
-        CIBW_ARCHS_MACOS: "x86_64"
+        CIBW_SKIP: "*i686* *win32* pp* *cp310*"
```

Comment:
`numpy` isn't releasing Python 3.10 wheels and building the numpy wheel manually results in a BLAS error.